composer.json

```diff
@@ -40,7 +41,8 @@
         "symfony/web-profiler-bundle": "^4.4 || ^5.1"
     },
     "suggest": {
-        "enqueue/elastica-bundle": "The bundle adds extra features to FOSElasticaBundle bundle. Aimed to improve performance."
+        "symfony/messenger": "For populating Elasticsearch indexes asynchronously and using significanly less resources.",
```

Comment:
We suggest the package but it's already a direct dependency :thinking:

Reply:
I don't think it was supposed to be a direct dependency.

Reply:
Should we remove the dependency?